### DISCUSSION TITLE
GitHub Docs: Error 503 certificate has expired

### DISCUSSION THREAD

**Discussion by renbaoshuo** [community/community#24676] [Apps, API and Webhooks]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/d/8/d82dd48c49b9e385dbfbfcfd280c7b9b522d52e5.png" data-download-href="/uploads/short-url/uQpk7poRGUOILEWXAkTrZ3CUQBv.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/47095648/181085998-c47edd02-9b37-4b27-b6c3-668c84ba3ca3.png" alt="image" data-base62-sha1="uQpk7poRGUOILEWXAkTrZ3CUQBv" width="690" height="408" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/d/8/d82dd48c49b9e385dbfbfcfd280c7b9b522d52e5_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">925×547 14.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>I can’t visit <a href="http://docs.github.com" rel="noopener nofollow ugc">docs.github.com</a> now, it throwed a 503 error and says “certificate has expired”.</p>

**Comment by nethgato** [comment, marked as answer]
<p>Hey, thanks for elevating this! We reported this on our Status Page:</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://www.githubstatus.com/incidents/22y5yl6gcc6b">
  <header class="source">
      <img src="https://dka575ofm4ao0.cloudfront.net/pages-favicon_logos/original/36420/akacZEQQfOBdc7ftyxJt" class="site-icon" width="32" height="32">

      <a href="https://www.githubstatus.com/incidents/22y5yl6gcc6b" target="_blank" rel="noopener">githubstatus.com</a>
  </header>

  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/62281706/181086001-0e6a2aba-71a6-4f38-a18e-7e6968505ab4.png" class="thumbnail onebox-avatar" width="120" height="120">

<h3><a href="https://www.githubstatus.com/incidents/22y5yl6gcc6b" target="_blank" rel="noopener">Incident with GitHub Docs</a></h3>



  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>…and it’s since been resolved.</p>
<p>Cheers!</p>
